src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -311,6 +325,33 @@ public List<DarCollectionSummary> getSummariesForRoleName(User user, String user
     return summaries;
   }
 
+  public DarCollectionSummary draftDarCollection(DarCollection sourceCollection) {
+    this.dataAccessRequestDAO.updateDraftByCollectionId(sourceCollection.getDarCollectionId(), true);
+    sourceCollection.getDars().values().forEach((d) -> {
+      Date now = new Date();
+      DataAccessRequestData newData = new Gson().fromJson(d.getData().toString(), DataAccessRequestData.class);
+      newData.setDarCode(null);
```

Comment:
I'm not sure we need to do this as this value is being deprecated.

Reply:
ahh right - I had done this as in the UI it only allows you to resubmit a draft if `isNil(darCode)` so without setting it to null it broke the UI. Should I remove this for now, even though it breaks the UI?

Reply:
Let's leave it in if it breaks the UI. I think the logic of `isNil(darCode)` is no longer valid. In this world, a DAR with a DarCollection (and therefore a dar code) is valid for resubmission. The condition for resubmission is that there are no elections created for it.